src/tests/ftest/erasurecode/online_rebuild_mdtest.yaml

```diff
@@ -58,4 +58,5 @@ mdtest:
       #- [EC_Object_Class, Minimum number of servers]
       - ["EC_2P2GX", 6]
       - ["EC_4P2GX", 8]
+      - ["EC_4P3GX", 9]
```

Comment:
also here for RF3 case with ECxP3, should we set dir_oclass to RP4 for this case?

Reply:
I am not sure how to expand this in yaml file. because there are different object classes which has different RP levels for directory now, so I leave it as RP3 for now.

A further think tests now at mostly killed two ranks(it assumed  2 parities  before) I suppose it should be smarter to kill different number of ranks according to different object classes. I suppose this is not an easy change for me, it will be nice that someone in CI could help improve this in the PR(or different PR later).

Reply:
ok maybe i misunderstood the purpose of this PR. i thought we want to test that EC xP3 will work with 3 concurrent fault domain failures?

Reply:
If we want to test `EC*PX` with `X` failures we'll need to make some utility changes. It shouldn't be too difficult but does affect several tests. I can help with that if needed

Reply:
Setting the dir_oclass to match the RF is also doable

Reply:
Also FWIW many of these tests with `P2` only test with 1 failure

Reply:
ok i think we need to improve that to properly test against data loss for rf2. for this PR though i think the changes made are sufficient. but still need a +1 from @daltonbohning

Reply:
I created ticket https://daosio.atlassian.net/browse/DAOS-16955 for following improvements